packages/flutter/lib/src/material/scrollbar.dart

```diff
@@ -334,7 +333,7 @@ class _MaterialScrollbarState extends RawScrollbarState<_MaterialScrollbar> {
       return Color.lerp(
         _scrollbarTheme.thumbColor?.resolve(states) ?? idleColor,
         _scrollbarTheme.thumbColor?.resolve(states) ?? hoverColor,
-        _hoverAnimationController.value,
+        0
```

Comment:
This would not allow users to disable it, it would just disable the animation entirely for everyone. This is likely not the way to fix https://github.com/flutter/flutter/issues/116212